src/Tool/Telemetry.cs

```diff
@@ -226,6 +232,27 @@ public static EventProperties AsTelemetryEvent(this ExecutedEventArgs info)
 
             return evt;
         }
+
+        public static EventProperties AsTelemetryEvent(this SimulatorPerformanceArgs info)
+        {
+            var evt = new EventProperties() { Name = "SimulatorPerformance".WithTelemetryNamespace() };
+
+            evt.SetProperty("SimulatorName".WithTelemetryNamespace(), info.SimulatorName);
+            evt.SetProperty("NQubits".WithTelemetryNamespace(), info.NQubits);
+            evt.SetProperty("Duration".WithTelemetryNamespace(), info.Duration);
```

Comment:
Looks like this will be necessary for the CI to succeed:
```suggestion
            evt.SetProperty("Duration".WithTelemetryNamespace(), info.Duration.ToString());
```